account_chart_update_multilang/wizards/wizard_chart_update.py

```diff
@@ -31,9 +31,10 @@ def _update_other_langs(self, templates):
             for key in self._diff_translate_fields(template, rec):
                 for lang in self._other_langs():
                     field = rec._fields[key]
-                    old_value = field._get_stored_translations(rec).get(
-                        rec.env.lang, "en_US"
-                    )
+                    stored_translation_rec = field._get_stored_translations(rec)
+                    if not stored_translation_rec:
+                        continue
+                    old_value = stored_translation_rec.get(rec.env.lang, "en_US")
```

Comment:
Didn't you mean this?

```suggestion
                    old_value = stored_translation_rec.get(rec.env.lang or "en_US") or ""
```